controllers/pkg/porch/condition/condition_test.go

```diff
@@ -106,3 +106,49 @@ func TestHasSpecificTypeConditions(t *testing.T) {
 		})
 	}
 }
+
+func TestPackageRevisionIsReady(t *testing.T) {
+	cases := map[string]struct {
+		conds    []porchv1alpha1.Condition
+		readyGates []porchv1alpha1.ReadinessGate
+		want bool
+	}{
+		"Ready": {
```

Comment:
Can you add some false conditions in here to show that it's only the one listed in the readiness gates that matters?